sources/au/victoria.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "http",
-    "data": "http://s3.amazonaws.com/data.openaddresses.io/cache/au-victoria.zip",
-    "website": "http://services.land.vic.gov.au/landchannel/content/vicmapdata?productID=1",
+    "data": "https://www.alantgeo.com.au/share/au-victoria.zip",
```

Comment:
Try uploading with this form:

https://results.openaddresses.io/upload-cache

It will generate a URL in our S3 that you can use, if you'd like.